### PR TITLE
[KO] Update `People/osu! team`

### PR DESCRIPTION
**Pull request by Sitoria**
some desc and person missed, re-paste and re-translated.

## check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] @sylphi 
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by Sitoria — COMMENTED** [review]
typo error

File: wiki/People/osu!_team/ko.md [hunk 1]
```diff
@@ -23,34 +21,34 @@ tags:
...
 | ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | 모딩 및 매핑 자문가 |
 | ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | osu!web 개발자, [공식 아티스트](https://osu.ppy.sh/beatmaps/artists/1) |
 | ::{ flag=JP }:: [notbakaneko](https://osu.ppy.sh/users/10751776) | osu!web 개발자 |
-| ::{ flag=DE }:: [Okoratu](https://osu.ppy.sh/users/1623405) | 모더레이터, 모딩 및 매핑 자문가 |
-| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | 모딩 및 매핑 자문가, Mappers' Guild 주최자, 공식 아티스트 지원 |
-| ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | 모딩 및 매핑 자문가 |
-| ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | 총괄 관리자, 위키 관리자 |
+| ::{ flag=DE }:: [Okoratu](https://osu.ppy.sh/users/1623405) | [모더레이터](/wiki/People/Global_Moderation_Team), 모딩 및 매핑 자문가 |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | [osu! 유튜브 채널](https://www.youtube.com/@osugame) 호스트, [Mappers' Guild](/wiki/Community/Mappers_Guild) 주최자, [공식 아티스트](/wiki/People/Featured_Artists) 지원 |
+| ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | [노미네이션 어세스먼트 팀](/wiki/People/Nomination_Assessment_Team) 리더 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:42]: a

File: wiki/People/osu!_team/ko.md [hunk 2]
```diff
@@ -23,34 +21,34 @@ tags:
...
+| ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | [모더레이터](/wiki/People/Global_Moderation_Team), [위키 관리자](/wiki/People/osu!_wiki_maintainers) |
 | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | 총괄 관리자 |
 | ::{ flag=AU }:: [smoogipoo](https://osu.ppy.sh/users/1040328) | osu! 개발자, osu!mania 애호가, 버그 관련 담당자 |
 | ::{ flag=PL }:: [spaceman_atlas](https://osu.ppy.sh/users/3035836) | osu! 개발자 |
 | ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) | osu! 인프라 개발자 |
 | ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | 총괄 관리자, 토너먼트 보조 |
 | ::{ flag=US }:: [tybug](https://osu.ppy.sh/users/12092800) | 총괄 관리자, 빗자루 청소부 |
-| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | [글로벌 모더레이션 팀](/wiki/People/Global_Moderation_Team) 리더, 비트맵 스포트라이트 매니저, Hivie의 비서 |
-| ::{ flag=SE }:: [Walavouchey](https://osu.ppy.sh/users/5773079) | 위키 관리자, 뉴스 에디터 |
+| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | [글로벌 모더레이션 팀](/wiki/People/Global_Moderation_Team) 리더, [비트맵 스포트라이트](/wiki/Beatmap_Spotlights) 메니저, Hivie의 비서 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:50]: d

File: wiki/People/osu!_team/ko.md [hunk 3]
```diff
@@ -61,26 +59,26 @@ tags:
...
 | 이름 | 역할 |
 | :-- | :-- |
-| ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | 총괄 관리자 및 도우미, Chibi 아티스트 |
-| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | 모더레이터, 커뮤니티 자문가 |
+| ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | 총괄 관리자 및 도우미, Chibi 아티스트. |
+| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | [모더레이터](/wiki/People/Global_Moderation_Team), 커뮤니티 자문가 |
 | ::{ flag=NO }:: [cYsmix](https://osu.ppy.sh/users/272870) | 오디오 디자이너, [공식 아티스트](https://osu.ppy.sh/beatmaps/artists/2) |
-| ::{ flag=US }:: [Daru](https://osu.ppy.sh/users/32480) | osu!painter |
-| ::{ flag=NZ }:: [deadbeat](https://osu.ppy.sh/users/128370) | 월드컵 주최자 및 관리자 |
+| ::{ flag=US }:: [Daru](https://osu.ppy.sh/users/32480) | osu!페인터 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:65]: copypasta issue

**Review by TicClick — APPROVED** [review]

**Pull request merged**